tools/libSwiftScan/CMakeLists.txt

```diff
@@ -47,11 +47,15 @@ add_link_opts(libSwiftScan)
 add_dependencies(compiler libSwiftScan)
 
 
-if(CMAKE_HOST_SYSTEM_NAME STREQUAL Windows)
-swift_install_in_component(TARGETS libSwiftScan
-  ARCHIVE DESTINATION "lib${LLVM_LIBDIR_SUFFIX}/swift/${SWIFT_SDK_${SWIFT_HOST_VARIANT_SDK}_LIB_SUBDIR}" COMPONENT compiler
-  LIBRARY DESTINATION "lib${LLVM_LIBDIR_SUFFIX}/swift/${SWIFT_SDK_${SWIFT_HOST_VARIANT_SDK}_LIB_SUBDIR}" COMPONENT compiler
-  RUNTIME DESTINATION "bin" COMPONENT compiler)
+if(CMAKE_SYSTEM_NAME STREQUAL Windows)
+  swift_install_in_component(TARGETS libSwiftScan
+    ARCHIVE DESTINATION "lib${LLVM_LIBDIR_SUFFIX}" COMPONENT compiler
+    LIBRARY DESTINATION "lib${LLVM_LIBDIR_SUFFIX}" COMPONENT compiler
+    RUNTIME DESTINATION "bin" COMPONENT compiler)
+
+  swift_install_in_component(DIRECTORY "${SWIFT_MAIN_INCLUDE_DIR}/swift-c/DependencyScan/"
+    DESTINATION "include/${SWIFT_SCAN_LIB_NAME}"
```

Comment:
Should this be `${CMAKE_INSTALL_INCLUDE_DIR}`?

Reply:
Yes but that is https://github.com/apple/swift/pull/64281 . The changes will conflict and I will add the `${CMAKE_INSTALL_INCLUDE_DIR}` in the PR I get to rebase on the other.